internal/aws/containerinsight/const.go

```diff
@@ -257,4 +261,18 @@ func init() {
 		ContainerRestartCount: UnitCount,
 		RunningTaskCount:      UnitCount,
 	}
+
+	PodPhaseMetricNames = map[corev1.PodPhase]string{
+		corev1.PodPending:   MetricName(TypePodStatus, "pending"),
```

Comment:
For these `MetricName`s, the key should be `TypePod` and the values are already available as consts [here](https://github.com/amazon-contributing/opentelemetry-collector-contrib/blob/aws-cwa-dev/internal/aws/containerinsight/const.go#L104).

So this would look something like `MetricName(TypePod, StatusRunning)` which will evaluate to `pod_` + `status_running` = `pod_status_running`

Reply:
Looks like "running" is the only one available there